README.md

```diff
@@ -158,6 +171,8 @@ writeCapabilities(
 
 ## Version History
 
+### 1.7 (2017-09-04)
+* Introduced `testProject` method, which queries Service Book for project repos.
```

Comment:
I would reword as something like: Introduces `ServiceBook` class, with `testProject` method to execute tests for all pipelines associated with the specified project name.

Reply:
Thanks - fixed.